dm/syncer/syncer.go

```diff
@@ -1847,7 +1853,9 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 			currentLocation = shardingReSync.currLocation
 			// if suffix>0, we are replacing error
 			s.isReplacingOrInjectingErr = currentLocation.Suffix != 0
-			s.locations.reset(shardingReSync.currLocation)
+			if ce := s.tctx.L().Check(zap.DebugLevel, "update locations"); ce != nil {
+				s.locations.reset(shardingReSync.currLocation)
```

Comment:
if it's not needed, maybe just remove it. doesn't help to locate problems if anything happened.

we can add it back later if necessiary